lib/helper/browser.go

```diff
@@ -18,6 +20,10 @@ import (
 //                                                                            //
 ////////////////////////////////////////////////////////////////////////////////
 
+var FirefoxPathMac = []string{`/Applications/Firefox.app/Contents/MacOS/firefox`}
+var FirefoxPathLinux = []string{`/usr/bin/firefox`}
+var FirefoxPathWindows = []string{`\Program Files\Mozilla Firefox\firefox.exe`}
```

Comment:
Let's lowercase these variables so they are not exported unless you feel there is a need to do so.